apps/src/templates/projects/GallerySwitcher.jsx

```diff
@@ -104,4 +100,11 @@ class GallerySwitcher extends Component {
     );
   }
 }
-export default Radium(GallerySwitcher);
+
+export default connect(state => ({
+  pageLocation: state.pageLocation
+}), dispatch => ({
+  selectGallery(gallery){
+    dispatch(selectGallery(gallery));
+  }
+}))(Radium(GallerySwitcher));
```

Comment:
connect actually has a short-form, where if you're just dispatch actions with the same props, I believe you can do this:
```
connect({state => ({
  pageLocation: state.pageLocation
}), { selectGallery });
```